### PR TITLE
fixed ch07/transform.qml

### PR DESCRIPTION
**Pull request by alexshen**

### CODE REVIEW

File: en/ch07/src/canvas/transform.qml [hunk 1]
```diff
@@ -39,13 +39,15 @@ Canvas {
...
 
         ctx.beginPath()
-        ctx.rect(-20, -20, 40, 40)
         ctx.translate(120,60)
+        ctx.rect(-20, -20, 40, 40)
         ctx.stroke()
 
         // draw path now rotated
         ctx.strokeStyle = "green"
+        ctx.beginPath();
```
Comment (jcelerier) [inline en/ch07/src/canvas/transform.qml]: not a Canvas expert but shouldn't there be a matching `ctx.closePath()` ?

Comment (alexshen) [reply en/ch07/src/canvas/transform.qml]: Not an expert either. `closePath` will close the current subpath which is not used for the next rendering, so I think we can simply `beginPath` without a paired `closePath`. But it doesn't hurt to add it in the case.

Comment (jryannel) [reply en/ch07/src/canvas/transform.qml]: closePath should not be required here, as there is no line required to from current point to end point. Need to verify later.

**Comment by jryannel** [comment]
This was fixed in another commit. Will close this

**Pull request closed without merging**